depthai_nodes/ml/parsers/detection.py

```diff
@@ -0,0 +1,137 @@
+import depthai as dai
+import numpy as np
+
+from ..messages.creators import create_detection_message
+from .base_parser import BaseParser
+
+
+class DetectionParser(BaseParser):
+    def __init__(
+        self,
+        output_layer_name: str = "",
+        conf_threshold: float = 0.5,
+        iou_threshold: float = 0.5,
+        max_detections: int = 100,
```

Comment:
Other detection parsers have `max_det` instead of `max_detections`. Stick with `max_det` because this name is used in nn archive